state/block.go

```diff
@@ -112,9 +111,9 @@ func (bs *blockState) GetBabeHeader(epoch uint64, slot uint64) (babe.BabeHeader,
 	return result, err
 }
 
-// GetLatestBlock returns the latest block available on blockState
-func (bs *blockState) GetLatestBlock() types.BlockHeader {
-	return bs.latestBlock
+// Header returns the latest block available on blockState
+func (bs *blockState) Header() *types.Header {
```

Comment:
I think this should be renamed LatestHeader() to be more descriptive

Reply:
Well, its not returning it from the DB, it returns a pointer; Latest implies a behavior that its not there.

Reply:
Updated as requested.